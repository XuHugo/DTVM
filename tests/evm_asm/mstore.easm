PUSH1 0xFF
PUSH1 0
MSTORE

PUSH1 0xFF
PUSH1 1
MSTORE
