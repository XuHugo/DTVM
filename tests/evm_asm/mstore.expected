status: success
error_code: 0
stack: []
memory: '0000000000000000000000000000000000000000000000000000000000000000FF00000000000000000000000000000000000000000000000000000000000000'
storage: {}
transient_storage: {}
return: ''
events: []
